protos/brbctpb/brbctpb.proto

```diff
@@ -0,0 +1,81 @@
+syntax = "proto3";
+
+package brbctpb;
+
+option go_package = "github.com/filecoin-project/mir/pkg/pb/brbctpb";
+
+import "mir/codegen_extensions.proto";
+import "net/codegen_extensions.proto";
+
+import "commonpb/commonpb.proto";
+
+// ============================================================
+// Events
+// ============================================================
+
+message Event {
+  option (mir.event_class) = true;
+
+  oneof type {
+    option (mir.event_type) = true;
+
+    BroadcastRequest request = 1;
+    Deliver          deliver = 2;
+  }
+}
+
+message BroadcastRequest {
+  option (mir.event) = true;
+
+  int64 id   = 1;
+  bytes data = 2;
+}
+
+message Deliver {
+  option (mir.event) = true;
+
+  int64 id   = 1;
+  bytes data = 2;
+}
```

Comment:
There is no need to repeat it for each `brb` implementation.
Make one `.proto` file to define the abstraction (`brb`) and then one separate `.proto` file for each of the 3 implementations (Bracha, CT, DXR) to define the implementation-specific message types.